### PR TITLE
Feature/wid 317

### PR DESCRIPTION
**Pull request by brampauwelyn**
### Added
- show editorial label if article is published on a partner website + when the setting is enabled in the widgetbuilder

---
Ticket: [https://jira.uitdatabank.be/browse/WID-317](https://jira.uitdatabank.be/browse/WID-317)

### CODE REVIEW

File: app/Widget/WidgetServiceProvider.php [hunk 1]
```diff
@@ -86,7 +86,9 @@ public function register(Container $pimple)
                 new TranslateTerm(
                     $pimple['translator']
                 ),
-                $pimple['translator']
+                $pimple['translator'],
+                $pimple['curatoren_api'],
+                $pimple['curatoren_api_test']
```
Comment (jdrieghe) [inline app/Widget/WidgetServiceProvider.php:91]: We only added one argument to the `TwigPreprocessor` constructor, why are we adding two here?

File: src/Widget/Twig/TwigPreprocessor.php [hunk 2]
```diff
@@ -387,6 +399,36 @@ public function preprocessArticles($linkedArticles, $settings)
...
+     * Preprocess the editorial label for search results
+     * @param string $cdbid
+     * @param string $settings
+     */
+    public function preprocessEditorialLabel($cdbid, $settings)
+    {
+        $articles = $this->curatorenClient->searchArticles($cdbid);
+
+        if (empty($articles['hydra:member'])) {
+            return;
```
Comment (jdrieghe) [inline src/Widget/Twig/TwigPreprocessor.php:412]: The return type hint of this method claims it always returns a string, yet we return an implicit `null` here.
Either:
- Change the return type to `string|null` and make the `return null` explicit here
- Return an empty string `return '';`

Comment (brampauwelyn) [reply src/Widget/Twig/TwigPreprocessor.php:413]: @jdrieghe jup thx for the remark, fixed it.

Comment (jdrieghe) [reply src/Widget/Twig/TwigPreprocessor.php]: Don't forget to replace the `return;`'s in this method with `return null;` now

**Pull request merged**